golang/go-cloud-run-hello-world/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/go-cloud-run-hello-world.iml" filepath="$PROJECT_DIR$/.idea/go-cloud-run-hello-world.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
This file was copied from the `go-hello-world` sample and the module reference updated to the dirname of this sample.